docs/Gemfile

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```

Comment:
Do we need this here? What's the point in having a commented out dependency?

Reply:
I was a left over comment, thanks for spotting it!